.flaskenv

```diff
@@ -1,2 +0,0 @@
-FLASK_DEBUG=1
-FLASK_APP=backend/app.py
```

Comment:
Sorry, I didn't notice it is pointing at `backend` subfolder, then I prefer to keep it at the top level but fixing the `tool.pdm.scripts` instead.